recipes/eleuther_eval.py

```diff
@@ -124,20 +124,25 @@ class EleutherEvalRecipe(EvalRecipeInterface):
     def __init__(self, cfg: DictConfig) -> None:
         self._cfg = cfg
 
-    def load_checkpoint(self, checkpointer_cfg: DictConfig) -> Dict[str, Any]:
+    def load_checkpoint(self, checkpointer_cfg: DictConfig, weights_only: bool = True) -> Dict[str, Any]:
```

Comment:
@joecummings can I get a review for this file? Trying to think about the best way to integrate the  quantization changes for eval

Reply:
This is mostly to load a quantized model for evaluation. The flow here is something like this:
- Finetune + Eval
- Quantize
- Eval with quantize to make sure quantized model is still performant
- Run inference with quantized model to make sure its not doing something crazy

Reply:
any updates on this? @kartikayk @joecummings